### PR TITLE
fix(thrift): Updated thrift configuration to adopt configurable max message size and max framesize

### PR DESCRIPTION
**Pull request by smrutis1**
Signed-off-by: Smruti Prakash Sahoo <smruti.sahoo@siemens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
Added a custom thrift configure to adopt configurable thrift max message size and max frame size. 

> * Which issue is this pull request belonging to and how is it solving it? (*#1493*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: 

This would appear with huge amount of data.

> Have you implemented any additional tests? - No

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

**Comment by smrutis1** [comment]
Default frame size: `16384000 [0xfa0000]`
Default max message size: `104857600 [0x6400000]`

File: libraries/lib-datahandler/src/main/resources/sw360.properties [hunk 1]
```diff
@@ -43,5 +43,9 @@
 ## first byte of the response is available.
 #backend.timeout.read = 600000
 
+## These two properties used to set max message size and frame size of thrift transport
+#backend.thrift.max.message.size=
```
Comment (ag4ums) [inline libraries/lib-datahandler/src/main/resources/sw360.properties:47]: Please include the default size(may be as comment) so the user can know the what is the current value and to waht value that can update

Comment (smrutis1) [reply libraries/lib-datahandler/src/main/resources/sw360.properties]: included default size

**Review by JaideepPalit — APPROVED** [review]
Code looks good to me.

**Review by ag4ums — APPROVED** [review]
working as expected

**Pull request merged**